packages/fx-core/templates/plugins/resource/cicd/github/README.md

```diff
@@ -0,0 +1,45 @@
+# How to use pre-cooked workflows on GitHub
+
+## Prerequisites
+- Teams app projects are version controlled by GitHub.
+- An M365 account. If you do not have M365 account, apply one from [M365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)
+- An Azure service princial with necessary permissions.
+
+## Steps
+After the pre-cooked workflows are scaffolded successfully, the following steps are expected to be performed:
+1. Configure necessary GitHub secrets for your workflows.
+1. Add the scaffolded yml files into GitHub by `git add`,`git commit` and `git push`.
+
```

Comment:
I suggested describe the steps like the following:

1. Commit and push your project source code to github remote repository, including the CICD yml files.
2. Click Actions in remote repository. (better to have a screenshot) 
![image](https://user-images.githubusercontent.com/25972542/157640955-a1a85d34-dabe-458a-ae66-781081e31d94.png)
3. Select the CICD action that you want to run.
4. (optional) If the action you selected is Provision or Publish to Teams App, you will need to config Gihub Secrets for your github action. Please see below [gihub secretes](github secretes)